package-lock.json

```diff
@@ -44408,6 +44385,18 @@
         "node": ">=6.0.0"
       }
     },
+    "packages/compass-aggregations/node_modules/@mongodb-js/shell-bson-parser": {
```

Comment:
Looks a bit suspicious that these got un-hoisted. Are there packages that still depend on the other version in the monorepo? 🤔

Reply:
Huh! Great catch, I didn't notice at all, and I don't know why npm did this, I literally only bumped the versions in the packages and ran "npm install".  And there aren't any other dependencies on these packages in Compass...

I undid this but will also keep an eye on this, this just seems like something that shouldn't be happening 🤔

Reply:
Might be some change in algo in some recent npm version maybe 🤔